nano framework build is failing in CI

Invoking build.cmd script ...
Build Amqp.Net Lite
Build solution: amqp-nanoFramework.sln
Build target: build
Build version: "2.3.0"
Build configuration: Debug
Build platform: Any CPU
Build run tests: false
Build NuGet package: false
NuGet: ".\Build\tools\NuGet.exe"
MSBuild: "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe"
dotnet: "C:\Program Files\dotnet\dotnet.exe"
MSTest: 
Build solution amqp-nanoFramework.sln
MSBuild auto-detection: using msbuild version '15.9.21.664' from 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin'.
All packages listed in packages.config are already installed.
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\nanoFramework\v1.0\NFProjectSystem.MDP.targets(718,5): error MSB4018: The "GenerateBinaryOutputTask" task failed unexpectedly. [C:\projects\amqpnetlite\nanoFramework\Amqp.nanoFramework.nfproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\nanoFramework\v1.0\NFProjectSystem.MDP.targets(718,5): error MSB4018: System.IO.IOException: The process cannot access the file 'C:\projects\amqpnetlite\bin\Debug\Amqp.nanoFramework\Amqp.nanoFramework.dll' because it is being used by another process. [C:\projects\amqpnetlite\nanoFramework\Amqp.nanoFramework.nfproj]

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -8,6 +8,8 @@ environment:
   matrix:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       FRAMEWORKS_TO_BUILD: REST_OF_FRAMEWORKS
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      FRAMEWORKS_TO_BUILD: NANOFRAMEWORK
 
 before_build:
 - ps: >-

